source/config/profileUpgradeSteps.py

```diff
@@ -15,6 +15,7 @@
 """
 
 from logHandler import log
+from typing import Dict

```

Comment:
For typing imports, we tend to use the following syntax, as the import list tends to grow quickly and this makes diffs easier in future

```suggestion
from typing import (
	Dict,
)
```